spec/fixtures/tmx/tmx_fixture.rb

```diff
@@ -16,7 +16,9 @@ def render
     end
 
     def wrap_placeholders(text)
-      text.gsub(/(\{\d+\})/) { "<ph>#{$1}</ph>" }
+      text.gsub(/(\{\d+\})/) do
+        "<ph type=\"x-smartling-placeholder\">#{$1}</ph>"
+      end
```

Comment:
Ahhhh so without this, then we would just wrap all our placeholders with just `<ph>` tags regardless